app/ducks/playbooks/index.js

```diff
@@ -0,0 +1,34 @@
+const initialState = {
+  playbooks: []
+};
+
+const types = {
+  PLAYBOOKS_ADD: 'PLAYBOOKS/PLAYBOOKS_ADD'
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+  case types.PLAYBOOKS_ADD:
```

Comment:
@marciobarrios this is actually changed by `prettier`... we should think about changing the settings for this.